be/src/olap/tablet.cpp

```diff
@@ -1636,4 +1641,13 @@ std::shared_ptr<MemTracker>& Tablet::get_compaction_mem_tracker(CompactionType c
     }
 }
 
+const TabletSchema& Tablet::tablet_schema() const {
+    std::shared_lock wrlock(_meta_lock);
+    const RowsetSharedPtr last_rowset = rowset_with_max_version();
```

Comment:
Currently rowset does not have tablet schema. So tablet schema == null

Reply:
the continue code process it. 
if (last_rowset == nullptr) {
        return _schema;
 }
that will use the schema from BE saves.